src/_attributes.scss

```diff
@@ -0,0 +1,86 @@
+@mixin text {
+	text-transform: uppercase;
+	text-align: center;
+	font-weight: bold;
+}
+
+@mixin footer-texts {
+	text-transform: uppercase;
+	text-align: left;
+	position: relative;
+	top: 2rem;
+	margin: 0;
+}
+
+@mixin footer-links {
+	background-color: $button;
+	width: 2rem;
+	height: 2rem;
+	background-repeat: no-repeat;
+	background-position: center;
+}
+
+@mixin background {
+	background-position: center;
+	background-repeat: no-repeat;
+}
+
+@mixin size ($width, $height) {
+	width: $width;
+	height: $height;
+}
+
+@mixin textarea-font {
+    font-weight: bold;
+    font-family: Arial;
+    font-size: 1rem;
+}
+
+$reduced: 0.8rem;
+
+/* colors */
+
+$white: #ffffff;
+$grey: #f2f2f2;
+$black: #000000;
+$red: #cc0000;
+$button: #90b2d0;
+$blue-hover: #90b1d0;
+$active: #b2b2b2;
+$subitem-active: #d4d0cd;
+$input-hover: #e4e4e4;
+$input-active: #d4d4d4;
+$radio-checked: #a4a4a4;
+$footer-hover: #799cbd;
+$footer-active: #6c94ba;
+$button-active: #98b4cf;
+$mobile-menu-border-top: #9ac2db;
+$mobile-menu-border-bottom: #678fa8;
+$mobile-menu-item-background: #81b3d2;
+$mobile-menu-hover-background: #799dbd;
+$mobile-menu-active-background: #6c95ba;
+$mobile-menu-active-text: #98b5cf;
+
+/* images */
+
+$header-background: url('img/header.png');
+$phone-logo: url('img/phone.png');
+$mail-logo: url('img/mail.png');
+$tick: url('img/tick.png');
```

Comment:
let's use the keyword to know what is it. 
E.g. 
"**logo**-something" is pretty good but "tick" is bad and not understandable.
"**icon**-something" is also good.

And let's use the same style for all the class names, I mean you should use or prefix or suffix.

Good:
$**icon**-twitter
$**logo**-phone
$**backgroung**-header

Bad:
$header-**background**
$phone-**logo**
$**logotype**-mobile